draft-ietf-tls-ctls.md

```diff
@@ -130,37 +130,69 @@ affected.  We do not show the structures which only change in this way.
 
 ## Record Layer
 
-The cTLS Record Layer assumes that records are externally framed
-(i.e., that the length is already known because it is carried in a UDP
-datagram or the like). Depending on how this was carried, you might
-need another byte or two for that framing. Thus, only the type byte
-need be carried and TLSPlaintext becomes:
+The only cTLS records that are sent in plaintext are handshake records
+(ClientHello and ServerHello/HRR).  The content type is therefore constant (it
+is always `handshake`), so we instead set the `content_type` field to a fixed
+cTLS-specific valud to distinguish cTLS plaintext records from encrypted
```

Comment:
Why can't we just use handshake here? It's distinguishable from the encrypted DTLS content types.

Reply:
nm. I see why.